components/datepicker/overview.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Overview
+title: Blazor DatePicker Overview | Telerik UI for Blazor
```

Comment:
This is the title displayed in the lefthand navigation. It must remain unchanged as it doesn't affect SEO in any way.
```suggestion
title: Overview
```